Friends

cheddartot opened the pull request.

cheddartot committed:
Added flash for all cases

Flashes a message on the page for all errors (exept friend already existing) and success on adding a friend. Doesn't actually add friends to the database yet.

cheddartot committed:
Friendships added

Actually creates friendships and adds them to the database, also verifies that friendship is valid and doesn't already exist before creation.

pgayee694 commented on app/routes.py:
```diff
@@ -82,10 +86,44 @@ def signup():
+    #TODO actually add friend group to db
+
+    friendship = models.Friends(friend1_id=friend1.id, friend2_id=friend2.id)
+    db.create_all()
```
you shouldn't need this here, this already runs in the app's init

cheddartot committed:
Removed unecessary create_all() call

removed said call and some old comments that were no longer needed.

cheddartot committed:
Moved  db calls to make code cleaner

moved a db.create_all() call in the _init_.py file to remove unecessary calls elsewhere.

cheddartot committed:
Merge branch 'master' into Friends
